docs/quickstart.asciidoc

```diff
@@ -101,77 +108,100 @@ spec:
 ----
 
+When you first create the Kubernetes cluster, there is no `HEALTH` status and the `PHASE` is empty. After the pod and service start-up, the `PHASE` turns into `Ready`, and `HEALTH` becomes `green`. The `HEALTH` status comes from {es}'s link:{ref}/cluster-health.html[cluster health API].
+
 [source,sh,subs="attributes"]
 ----
 NAME          HEALTH    NODES     VERSION   PHASE         AGE
-quickstart    green     1         {version}     Ready         1m
+quickstart              1         {version}               1s
 ----
 
-When you create the cluster, there is no `HEALTH` status and the `PHASE` is empty. After a while, the `PHASE` turns into `Ready`, and `HEALTH` becomes `green`. The `HEALTH` status comes from link:{ref}/cluster-health.html[Elasticsearch's cluster health API].
-
-One Pod is in the process of being started:
+While the {es} pod is in the process of being started it will report `Pending` as seen via link:https://kubernetes.io/docs/reference/kubectl/generated/kubectl_get/[`get`]:
 
 [source,sh]
 ----
 kubectl get pods --selector='elasticsearch.k8s.elastic.co/cluster-name=quickstart'
 ----
 
+Which will output similar to:
+
 [source,sh]
 ----
 NAME                      READY   STATUS    RESTARTS   AGE
-quickstart-es-default-0   1/1     Running   0          79s
+quickstart-es-default-0   0/1     Pending   0          9s
 ----
 
-Access the logs for that Pod:
+During and after start-up, up that pod's link:https://kubernetes.io/docs/reference/kubectl/generated/kubectl_logs/[`logs`] can be accessed:
 
 [source,sh]
 ----
 kubectl logs -f quickstart-es-default-0
 ----
 
+Once the pod has finished coming up, our original link:https://kubernetes.io/docs/reference/kubectl/generated/kubectl_get/[`get`] request will now report:
+
+[source,sh,subs="attributes"]
+----
+NAME          HEALTH    NODES     VERSION   PHASE         AGE
+quickstart    green     1         {version}     Ready         1m
+----
+
 [float]
-=== Request Elasticsearch access
+=== Request {es} access
 
-A ClusterIP Service is automatically created for your cluster:
+A `ClusterIP` Service is automatically created for your cluster as seen via link:https://kubernetes.io/docs/reference/kubectl/generated/kubectl_get/[`get`]:
 
 [source,sh]
 ----
 kubectl get service quickstart-es-http
 ----
 
+Which will output similar to:
+
 [source,sh]
 ----
 NAME                 TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
 quickstart-es-http   ClusterIP   10.15.251.145   <none>        9200/TCP   34m
 ----
 
+In order to make requests to the link:{ref}/rest-apis.html[{es} API]:
+
 . Get the credentials.
 +
-A default user named `elastic` is created by default with the password stored in a Kubernetes secret:
+By default, a user named `elastic` is created with the password stored inside a link:https://kubernetes.io/docs/concepts/configuration/secret/[Kubernetes secret]. This default user can be disabled if desired, see <<{p}-users-and-roles>> for more information.
```

Comment:
We tried to avoid the use of "see" in the past: https://github.com/elastic/cloud-on-k8s/pull/5315
```suggestion
By default, a user named `elastic` is created with the password stored inside a link:https://kubernetes.io/docs/concepts/configuration/secret/[Kubernetes secret]. This default user can be disabled if desired, refer to <<{p}-users-and-roles>> for more information.
```